samples/custom-hostname-instance/getInstanceHostname.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Prints a hostname of the GCP VM instance.
```

Comment:
GCP or Google Cloud Platform is not an approved name. It's Google Cloud.
Prints the hostname of the Google Cloud VM instance.

Reply:
Oh, that's a good point. Thanks!